### PR TITLE
Deprecate FailOnMissionControlTestFailure

### PR DESCRIPTION
**Pull request by rokonec**

### CODE REVIEW

File: src/Microsoft.DotNet.Helix/Sdk/CheckHelixJobStatus.cs [hunk 1]
```diff
@@ -1,10 +1,9 @@
+using Microsoft.Build.Framework;
+using Newtonsoft.Json;
```
Comment (tkapin) [inline src/Microsoft.DotNet.Helix/Sdk/CheckHelixJobStatus.cs]: Are these two newly added dependencies really needed? I don't see from the change why they would be needed.

Comment (sunandabalu) [reply src/Microsoft.DotNet.Helix/Sdk/CheckHelixJobStatus.cs]: They are not newly added, they just got re-arranged.

Comment (tkapin) [reply src/Microsoft.DotNet.Helix/Sdk/CheckHelixJobStatus.cs]: Ah, correct - still not sure if Newtonsoft.Json is needed though, will have to open this in VS.

Comment (rokonec) [reply src/Microsoft.DotNet.Helix/Sdk/CheckHelixJobStatus.cs]: I have used remove-and-sort of VS 2019.
Do we have a coding standard on 'using' order?

Comment (MattGal) [reply src/Microsoft.DotNet.Helix/Sdk/CheckHelixJobStatus.cs]: There's a few of them, but we tend to defer to what VS's default reordering does.  Alphabetical with System. separated by a line is good IMO.

**Pull request merged**